src/server.h

```diff
@@ -1651,6 +1708,10 @@ struct valkeyServer {
     list *clients_pending_write;           /* There is to write or install handler. */
     list *clients_pending_read;            /* Client has pending read socket buffers. */
     list *replicas, *monitors;             /* List of replicas and MONITORs */
+    rax *replicas_waiting_psync;           /* Radix tree using rdb-client id as keys and rdb-client as values.
```

Comment:
Consider matching the existing client rax index?

```suggestion
    rax *rdbonly_clients_index;           /* Radix tree using rdb-client id as keys and rdb-client as values.
```

Reply:
The goal is more important than the specific structure used to store it. For the sake of readability, I prefer to keep the existing name.